content/providers/03-community-providers/07.voyage-ai.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Voyage AI Provider
+description: Learn how to use the Voyage AI provider.
+---
+
+# Voyage AI Provider
+
+[patelvivekdev/voyage-ai-provider](https://github.com/patelvivekdev/voyageai-ai-provider) is a community provider that uses [Voyage AI](https://www.voyageai.com) to provide Embedding support for the AI SDK.
+
+## Setup
+
+The Ollama provider is available in the `voyage-ai-provider` module. You can install it with
+
+<Tabs items={['pnpm', 'npm', 'yarn']}>
+  <Tab>
+    <Snippet text="pnpm add voyage-ai-provider" dark />
+  </Tab>
+  <Tab>
+    <Snippet text="npm install voyage-ai-provider" dark />
+  </Tab>
+  <Tab>
+    <Snippet text="yarn add voyage-ai-provider" dark />
+  </Tab>
+</Tabs>
+
+## Provider Instance
+
+You can import the default provider instance `voyage ` from `voyage-ai-provider`:
+
+```ts
+import { voyage } from 'voyage-ai-provider';
+```
+
+If you need a customized setup, you can import `createVoyage` from `voyage-ai-provider` and create a provider instance with your settings:
+
+```ts
+import { createVoyage } from 'voyage-ai-provider';
+
+const voyage = createVoyage({
+  baseURL: 'https://api.voyageai.com/v1',
+  apiKey: process.env.VOYAGE_API_KEY,
+});
+```
+
+You can use the following optional settings to customize the Ollama provider instance:
```

Comment:
why does it say ollama everywhere

Reply:
> why does it say ollama everywhere
Sorry, I used Ollama as a reference to write that and forgot to update it.